config: add `include` field for config files (#11584)
tarantool/tarantool: changelogs/unreleased/gh-11583-config-files-include.md

## feature/config

> * Introduced an `include` config field for config files (gh-11510).

Reviewer: ```suggestion
* Introduced a new `include` field for configuration files (gh-11510).
```

Author: Thanks, fixed.
